tests/test_puller.py

```diff
@@ -21,5 +22,11 @@ def test_get_new_unordered(self):
         expectation = [{"message": "Moscow","update_id": 5}, {"message": "message","update_id": 4}]
         self.assertEqual(get_new(updates, 3), expectation)
 
+    def test_add_queue(self):
+        updates = [{"message": "Moscow","update_id": 5}, {"message": "/start","update_id": 1},{"message": "/weather","update_id": 3}, {"message": "London, UK","update_id": 2},{"message": "message","update_id": 4}]
+        queue = deque()
+        buffer = get_new(updates, 4)
```

Comment:
You don't need to create a buffer - you can just pass its output straight away.

The point of unit tests is to test a single function with its input and output. Hence we need to call as few our functions as possible.